### PR TITLE
Remove player names from VoN Overlay

### PR DESCRIPTION
**Pull request by thebonbon**
This change removes the player names from the VoN UI (talking and radio transmissions).
This is in addition to https://github.com/EveronLife/EveronLife/pull/191 (Removed nametags over head)

Also adds:
+ Simple guide to enable names again if not wanted.

### CODE REVIEW

File: src/Worlds/DebugWorld/DebugWorld_DebugWorldActions.layer [hunk 1]
```diff
@@ -71,4 +71,12 @@ GenericEntity {
...
   }
  }
+}
+$grp GenericEntity : "{9B6B61BB3FE3DFB0}Prefabs/Items/Equipment/Radios/Radio_ANPRC77.et" {
+ Radio_ANPRC77_5CAB08C53F86CCBB {
+  coords 101.008 0.001 134.949
+ }
+ Radio_ANPRC77_5CAB08C53FE4BA26 {
+  coords 100.371 0.001 134.926
+ }
```
Comment (Arkensor) [inline src/Worlds/DebugWorld/DebugWorld_DebugWorldActions.layer:81]:
```suggestion
```

Comment (Arkensor) [reply src/Worlds/DebugWorld/DebugWorld_DebugWorldActions.layer:81]: I would remove the radios again from debug world and if needed for testing rather add radios to the default loadout later on?

File: src/UI/Layouts/HUD/VON/VoNOverlay.layout [hunk 2]
```diff
@@ -0,0 +1,14 @@
+FrameWidgetClass {
+ {
+  VerticalLayoutWidgetClass "{59389903A80C9C04}" {
+   Prefab "{59389903A80C9C04}"
+   {
+    VerticalLayoutWidgetClass "{00000000AB3C9068}" {
+     Prefab "{00000000AB3C9068}"
+     "Is Enabled" 0
+     "Is Visible" 0
```
Comment (Arkensor) [inline src/UI/Layouts/HUD/VON/VoNOverlay.layout:9]: Disable both? Why not only "Is Visible"?

File: src/Worlds/DebugWorld/DebugWorld_DebugWorldActions.layer (hunk 1 above)
Comment (thebonbon) [reply src/Worlds/DebugWorld/DebugWorld_DebugWorldActions.layer:81]: Sure we can remove them again. Not sure if we should add radios to the default loadout, maybe if we can get a debug world only player loadout?

File: src/UI/Layouts/HUD/VON/VoNOverlay.layout (hunk 2 above)
Comment (thebonbon) [reply src/UI/Layouts/HUD/VON/VoNOverlay.layout]: Should only be not visible but enabled. Saving Layouts does not always really save it.

**Pull request merged**